docs/site/migration/tables/lb4-only-commands.html

```diff
@@ -0,0 +1,150 @@
+<head>
+  {% include analytics.html %}
+</head>
+
+<table width="750">
+  <thead>
+    <tr>
+      <th width="200">Commands</th>
+      <th>Description</th>
+      <th>Reason for difference from LB3</th>
+    </tr>
+  </thead>
+  <tbody>
+
+    <tr>
+      <td>
+        <code>lb4 repository</code>
+      </td>
+      <td> Create a repository for a model
+      </td>
+      <td> A new concept in LB4. A model's repository defines the CRUD operations. In LB3, the model defined the CRUD operations.
```

Comment:
Just wondering whether it would be worth it to add links to the LB4 pages, like a link to https://loopback.io/doc/en/lb4/Repositories.html, so people can learn more about it there?

Reply:
@nabdelgadir , I will add one for controllers as well.